### PR TITLE
Migrate email report settings for users

### PR DESCRIPTION
**Pull request by agrzywinski**
http://manage.dimagi.com/default.asp?162237
@czue @modonnell729
Please don't merge it until we want to move from v1 to commcare

### CODE REVIEW

File: custom/ewsghana/api.py [hunk 1]
```diff
@@ -132,6 +166,14 @@ class EWSApi(APISynchronization):
     ]
     PRODUCT_CUSTOM_FIELDS = []
 
+    def apis(self):
```
Comment (czue) [inline custom/ewsghana/api.py]: @agrzywinski perhaps we can just leave the extensions commented out but merge everything else? that way there will be less likelihood of merge conflicts when it is time to turn them on.

**Comment by czue** [comment]
also build is failing
```
======================================================================
FAIL: test_parse_monthlyreports_json (custom.ewsghana.tests.test_apis.ApisTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/custom/ewsghana/tests/test_apis.py", line 98, in test_parse_monthlyreports_json
    self.assertEqual(1, monthly_report.day_of_month)
AssertionError: 1 != None
```

File: custom/ewsghana/api.py [hunk 2]
```diff
@@ -10,7 +11,7 @@
 from jsonobject.properties import StringProperty, BooleanProperty, ListProperty, IntegerProperty, ObjectProperty
 from custom.ilsgateway.api import ProductStock, StockTransaction
 from jsonobject import JsonObject
-from custom.logistics.api import LogisticsEndpoint, APISynchronization, MigrationException
+from custom.logistics.api import LogisticsEndpoint, APISynchronization, MigrationException, ApiSyncObject
```
Comment (dimagimon) [inline custom/ewsghana/api.py]: F401 'ApiSyncObject' imported but unused

**Comment by agrzywinski** [comment]
@czue

**Comment by czue** [comment]
I think the build fail is related to https://github.com/dimagi/commcare-hq/pull/6409 but will wait for that to be merged and rebuild

**Comment by czue** [comment]
@agrzywinski confirming is fine to merge now once tests pass?

**Comment by agrzywinski** [comment]
@czue Yes, I comment out ApiSyncObjects for scheduled reports in EWSApi so it should not start for now in migration.

**Comment by NoahCarnahan** [comment]
@czue @agrzywinski, It looks like the test that is failing in this PR is a different test than the one I fixed in #6409.

**Pull request closed without merging**